src/fparser/two/Fortran2008.py

```diff
@@ -1578,6 +1579,54 @@ def match(string):
         )
 
 
+class Procedure_Stmt(Procedure_Stmt_2003):  # R1206
```

Comment:
I note this file is now well over 1K lines. We really don't want to carry on towards the situation we have with `Fortran2003.py`. Perhaps we should have a new `Fortran2003` directory and break out the classes into separate files within it. If we agree, that can be a separate PR as it's going to be big and disruptive...

Reply:
Yes I agree, if we can work out how to do it with the class generation code.